.gitmodules

```diff
@@ -0,0 +1,4 @@
+
+[submodule "include"]
+	path = include/spectroscope-proto
+	url = https://github.com/HikaruG/spectroscope-proto
```

Comment:
I know I had used this pattern previously on various project, but I've come to believe it's a bad pattern and you should just put these proto files under `proto` in this project.